Source/DafnyCore/Compilers/CSharp/Compiler-Csharp.cs

```diff
@@ -1254,6 +1255,27 @@ bool InvalidType(Type ty) => (ty.AsTypeParameter != null && ty.AsTypeParameter.E
       return cw;
     }
 
+    void GenerateIsMethod(RedirectingTypeDecl declWithConstraints, ConcreteSyntaxTree wr) {
```

Comment:
I think it would've been better if all the code generation code had all been written without the 'writer' style, so the signature for this method would be `ConcreteSyntaxTree GenerateIsMethod(RedirectingTypeDecl declWithConstraints)`.

It's never too late to start!